tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
```

Comment:
With no `target` specified, the default is `"ES3"`, meaning TS will transpile everything down to ES3 syntax.  It'd probably be better to set the target higher (I think even "TSNext" would be safe) and leave the transpiling to `babel-preset-env`.

Reply:
Thanks for the hint! Apparently, the `target` doesn't seem to make any difference when using Babel's TypeScript plugin, though. The build output was exactly the same no matter whether I set `"target": "es2018"` or not.

Reply:
It's still safer to explicitly set the target, IMO. Since we're compiling using Babel, it's safe to set it as `esnext`.

Reply:
Done, although I chose `es2018` (as `@babel/preset-env` wouldn't compile down non-standardised features).